doc/source/docs/user_guide/mapping.rst

```diff
@@ -23,19 +23,21 @@ Loading some example data:
 
 .. ipython:: python
 
-    world = geopandas.read_file(geopandas.datasets.get_path('naturalearth_lowres'))
-    cities = geopandas.read_file(geopandas.datasets.get_path('naturalearth_cities'))
+    import geodatasets
+
+    chicago = geopandas.read_file(geodatasets.get_path("geoda chicago health"))
+    groceries = geopandas.read_file(geodatasets.get_path("geoda groceries"))
 
 You can now plot those GeoDataFrames:
 
 .. ipython:: python
 
-    # Examine country GeoDataFrame
-    world.head()
+    # Examine chicago GeoDataFrame
```

Comment:
```suggestion
    # Examine the chicago GeoDataFrame
```
debated if this should read chicago health, but the health information isn't used, but rather population, so that's maybe less clear.

Reply:
the health data here is not very relevant